frontend/src/app/modules/common/colors/colors-autocompleter.component.ts

```diff
@@ -27,10 +27,11 @@
 // ++
 
 import {Component, ElementRef, OnInit} from '@angular/core';
-import {DynamicBootstrapper} from "core-app/globals/dynamic-bootstrapper";
 import {Highlighting} from "core-components/wp-fast-table/builders/highlighting/highlighting.functions";
 import {I18nService} from "core-app/modules/common/i18n/i18n.service";
 
+export const colorsAutocompleterSelector = 'colors-autocompleter';
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/3b1eae5a64159f8a044579a25870c772186dbd3f/frontend/src/app/modules/common/colors/colors-autocompleter.component.ts#L34"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>